### PR TITLE
Date column changed

### PR DESCRIPTION
**Pull request by Shriram-Patil-Arista**
Screenshot from 2024-02-28 12-49-38

As we can see, the filtering is working without throwing error

### CODE REVIEW

File: uvm/js/common/util/Converter.js [hunk 1]
```diff
@@ -23,6 +23,18 @@ Ext.define('Ung.util.Converter', {
         return Ext.util.Format.date(date, 'timestamp_fmt'.t());
     },
 
+    convertFn: function (value) {
```
Comment (mayur-dhande) [inline uvm/js/common/util/Converter.js:26]: Can you please add some documentation for this function ?

Comment (cblaise-untangle) [reply uvm/js/common/util/Converter.js:26]: Also, let's rename it to something like "convertDate" instead of this generic name.

Comment (Shriram-Patil-Arista) [reply uvm/js/common/util/Converter.js]: changed the convertFn name to convertDate

**Review by cblaise-untangle — APPROVED** [review]

File: uvm/js/common/util/Converter.js (hunk 1 above)
Comment (Shriram-Patil-Arista) [reply uvm/js/common/util/Converter.js]: @mayur-dhande  will do that in next PR.

**Pull request merged**